app/controllers/api/account.php

```diff
@@ -897,9 +898,9 @@
     ->inject('project')
     ->inject('dbForProject')
     ->inject('locale')
-    ->inject('events')
-    ->inject('mails')
-    ->action(function (string $userId, string $email, string $url, Request $request, Response $response, Document $user, Document $project, Database $dbForProject, Locale $locale, Event $events, Mail $mails) {
+    ->inject('queueForEvents')
+    ->inject('queueForMails')
```

Comment:
I don't see these resources being declared in the init.php file

Reply:
they are